docs/overrides/partials/integrations/analytics/custom.html

```diff
@@ -0,0 +1,77 @@
+<script src="https://cdn.amplitude.com/libs/analytics-browser-1.9.1-min.js.gz"></script>
+<script type="text/javascript">
+
+const amplitudeKey = "8ab9ae57b486baa5d4ab626559fd8106"
```

Comment:
If anyone knows a good way for me to switch this between stg/prd via configuration of some kind, I'd love suggestions

Reply:
I don't know if mkdocs or material-mkdocs supports custom variables, this is probably fine for now